dataset.go

```diff
@@ -299,3 +299,19 @@ func (set dataset) Strings() []string {
 	}
 	return res
 }
+
+// ColumnStrings returns string values of selected columns of a provided dataset
```

Comment:
```suggestion
// ColumnStrings returns string values of selected columns of a provided dataset
// in case no cols provided, all columns will be selected 
```
or something similar 🙃